eventhub/client.go

```diff
@@ -103,19 +105,27 @@ func SubscribePartitions(ctx context.Context, sess *amqp.Session, name, group st
 // PutTokenContinuously writes token first time in blocking mode and returns
 // maintaining token updates in the background until stopCh is closed.
 func (c *Client) PutTokenContinuously(
-	ctx context.Context,
 	audience string,
 	token string,
+	cred *common.Credentials,
 	stopCh chan struct{},
 ) error {
-	if err := c.PutToken(ctx, audience, token); err != nil {
+	if err := c.PutToken(context.Background(), audience, token); err != nil {
 		return err
 	}
 	go func() {
+		ticker := time.NewTimer(30 * time.Minute) // 30min is half of 1hour
```

Comment:
Seems like the default token lifetime value is 1 hour: https://docs.microsoft.com/en-us/azure/active-directory/active-directory-configurable-token-lifetimes

I think we need to reset ticker each time we successfully put token with 1 hour duration minus 5-15 min as a safe buffer.

Reply:
🆗 I will set 50min.

we can set [here](https://github.com/goautomotive/iothub/blob/5a80cb9a250d0ea0b6102bbc4878560ec307c65e/common/sas.go#L53), using `duration`.
I think lifetime value haven't default value, Is this correct?